source/docs/software/driverstation/driver-station-best-practices.rst

```diff
@@ -1,77 +1,97 @@
+
+## Prior to leaving for the competition
+
+Items in this section require internet connectivity and should be done prior to arriving at the event.
+
+1. Install all Windows updates, then test the driver station with your robot. Keep running Windows Update until no updates show as pending.
+#. Pause updates. Navigate to Start -> Settings -> Windows Update, then set Pause updates to cover the time of the competition.
+#. Change "Active Hours" for Windows Updates to prevent updates from installing during competition hours. This helps if you forget to pause updates.  Navigate to Start -> Settings -> Windows Update -> Advanced options, then select Active Hours. Be sure to take timezone differences at your competition location into account.
+#. Bring a trusted USB and Ethernet cable for use connecting to the roboRIO.
 
 ## At The Competition
 
 1. Turn off Windows firewall using :ref:`these instructions <docs/networking/networking-introduction/windows-firewall-configuration:Disabling Windows Firewall>`.
-2. Turn off the Wi-Fi adapter, either using the dedicated hardware Wi-Fi switch or by disabling it in the Adapter Settings control panel.
-3. Charge the driver station when it’s in the pit.
-4. Remove login passwords or ensure everyone on the drive team knows the password. You'd be surprised at how often drivers arrive at the field without knowing the password for the laptop.
-5. Ensure your LabView code is deployed permanently and set to "run as startup", using the instructions in the LabView Tutorial. If you must deploy code every time you turn the robot on, you’re doing it wrong.
-6. Limit web browsing to FRC related web sites. This minimizes the chance of getting malware during the competition.
-7. Don't plan on using internet access to do software updates. There likely won't be any in the venue, and hotel Wi-Fi varies widely in quality. If you do need updates, contact a Control System Advisor in the pit.
+#. Turn off the Wi-Fi adapter, either using the dedicated hardware Wi-Fi switch or by using the Wi-Fi control in Quick Settings.
+#. Charge the driver station when it’s in the pit.
+#. Ensure everyone on the drive team knows the password. You'd be surprised at how often drivers arrive at the field without knowing the password for the laptop.
+#. LabView users: Ensure your LabView code is deployed permanently and set to "run as startup", using the instructions in the LabView Tutorial. If you must deploy code every time you turn the robot on, you’re doing it wrong.
+#. Limit web browsing to FRC related web sites. This minimizes the chance of getting malware during the competition.
+#. Don't plan on using internet access at the event to make software updates. There likely won't be any in the venue, and hotel Wi-Fi varies widely in quality. If you do need updates, contact a Control System Advisor in the pit.
```

Comment:
Maybe this should move to before competition. If you're reading this at competition, it's too late.